haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -206,21 +206,12 @@ newtype VotingPeriod = VotingPeriod { unVotingPeriod :: Natural}
 instance HasAnnotation VotingPeriod where
   annOptions = baseDaoAnnOptions
 
-newtype MaxChangePercent = MaxChangePercent Natural
```

Comment:
These newtypes for `MaxChagePercent` and `ChangePercent` exists only because we might want to have different `IsConfigDescExt` instance for them to set them in the `Config` during tests. It appears that we don't need those instances as of now. So it might be fine.

Reply:
Oh I see, I missed that point.

Since using the `IsConfigDescExt` will require to build the datatype (whichever we chose) I think this can be also used well by using `fillConfig` with ``percentageToFractionNumerator` (or something similar) if we ever need to :thinking: 

Anyway since as you said we don't need this now I would keep this change as it's closer to how the LIGO types are defined.